employee_jarsa.sql

```diff
@@ -3,13 +3,64 @@
 --       You can create database locally to test it.
 --       Consider add ';' at end sentence.
 
-CREATE TABLE employee (
+create table employee_department(
+id int auto_increment,
+namee varchar(20),
+description text,
+primary key(id)
 );
 
-CREATE TABLE employee_department (
+create table employee (
+id int auto_increment,
+first_name varchar(30),
+last_name varchar(30),
+id_department int,
+primary key(id),
+foreign key(id_department) references employee_department(id)
 );
 
-CREATE TABLE employee_hobby (
+insert into employee_department(id,namee,description) values(0,'Ventas','Creado para vender');
```

Comment:
You have an error in this inserts because in the employee_department table creation you have a syntax error, please check it